Cargo.toml

```diff
@@ -82,4 +82,4 @@ rev = "81edfcd908be8d3b2cd763cf2dc76aedbb22286b"
 # Fixes formatting of u32, however, it causes a link error due to u32-div
 # Which in turn is fixed (in config.toml) by:
 #     build-std-features = ["compiler-builtins-mangled-names"]
-ufmt = { git = "https://github.com/mrk-its/ufmt.git", rev = "e6e574625e89bce9a5cb907eae43bfd45672f768" }
```

Comment:
If `ufmt` v0.2 fixes the `u32` formatting, we no longer need the patch section.

Reply:
Removed